src/screenComponents/combatManeuver.cpp

```diff
@@ -29,6 +37,24 @@ GuiCombatManeuver::GuiCombatManeuver(GuiContainer* owner, string id)
 
 void GuiCombatManeuver::onDraw(sf::RenderTarget& window)
 {
+	if (slider->getValue().x ==0 && slider->getValue().y==0)
+	{
+		float strafe_amount=0.0;
+		float boost_amount=0.0;
+
+		if (sf::Keyboard::isKeyPressed(combat_left_keycode)){
+			strafe_amount=-1.0;
+		}
+		if (sf::Keyboard::isKeyPressed(combat_right_keycode)){
+			strafe_amount=1.0;
+		}
+		if (sf::Keyboard::isKeyPressed(combat_boost_keycode)){
+			boost_amount=1.0;
+	}
+		my_spaceship->commandCombatManeuverStrafe(strafe_amount);
+	my_spaceship->commandCombatManeuverBoost(boost_amount);
```

Comment:
You should only call the "command" functions if the strafe/boost amount changed, else you are generating a constant stream of network traffic (as the command functions generate network traffic)

Reply:
I see. Fixed in efdcba0b1542522e313e424c023238c1a41fe82a